components/generic_modal.scss

```diff
@@ -243,6 +244,9 @@
 }
 
 .GenericModal__header {
+    display: flex;
```

Comment:
Instead of adding `display:flex` property in the existing class would be better to use a new class in addition to this. This has the potential of adding UI bugs.

Reply:
Good point, thanks. I added these to css file specific to `Browse Channel` modal